src/templates/app/user/metatrader.jsx

```diff
@@ -303,7 +303,7 @@ const Metatrader = () => (
                                         </TypeGroup>
                                     </div>
                                     <p id='new_account_msg' className='notice-msg center-text invisible' />
-                                    <p id='new_account_financial_authenticate_msg' className='invisible notice-msg hint'>{it.L('You may proceed but please [_1]authenticate[_2] your account within 10 days to continue trading, otherwise your account will be disabled and any open positions will be closed.', `<a href="${it.url_for('user/authenticate')}">`, '</a>')}</p>
+                                    <p id='new_account_financial_authenticate_msg' className='invisible notice-msg hint'>{it.L('You may proceed but please authenticate your account within 10 days to continue trading, otherwise your account will be disabled and any open positions will be closed.')}</p>
```

Comment:
Why was the link removed?

Reply:
this message shown before mt5 signup, ergo, authenticate page is not available